pkg/service/backup/worker_snapshot.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/scylladb/scylla-manager/v3/pkg/util/timeutc"
 )
 
-func (w *worker) Snapshot(ctx context.Context, hosts []hostInfo, limits []DCLimit) (err error) {
+func (w *worker) Snapshot(ctx context.Context, hosts []hostInfo, limits []DCLimit) (err error) { // nolint: dupl
```

Comment:
you lazy bastard...
But frankly it's much better than customizing the insides

Reply:
I just wanted to include a quick fix so that it can be included in the release!

Reply:
I fixed it in 3e9a9de3.